README.md

```diff
@@ -9,3 +9,25 @@ It detects whether the container environment contains secret references (`secret
 1. It prefixes the target containers command with `<path/to/volume>/secrethub run --`.
 
 This project is based on and heavily inspired by [Berglas's Kubernetes Mutating Webhook](https://github.com/GoogleCloudPlatform/berglas/tree/v0.5.1/examples/kubernetes).
+
+## Deploy the Webhook
+
+The simplest method to deploy the webhook is in a serverless function. Below we outline the steps to take to deploy the webhook to a Google Cloud Function.
+We're also [working on](https://github.com/secrethub/secrethub-kubernetes-mutating-webhook/pull/2) a way to deploy the webhook in the Kubernetes cluster itself.
+
+You can deploy the webhook to a Google cloud function using the following steps:
+
```

Comment:
Don't you have to `git clone` first? Or where do I have to execue this?

Reply:
Yes, you have to clone first. I thought this is so obvious that we can leave it out. But because you're asking this question I'll add this step to make it extra clear.